yb-voyager/cmd/getDataMigrationReportCommand.go

```diff
@@ -283,7 +283,8 @@ func getFinalRowCount(row rowData) int64 {
 
 func init() {
 	getCommand.AddCommand(getDataMigrationReportCmd)
-	registerCommonGlobalFlags(getDataMigrationReportCmd)
```

Comment:
I understand we want to get rid of "yes" and "send-diagnostics" for this command, but we seem to be duplicating the export-dir flag definition in multiple places. (cutover related commands as well have their separate definition). 

How about a function for registering the export-dir flag?

Reply:
Yes makes sens